Nur in zaphistory-0.0.3: libvdr-zaphistory.so.
Nur in zaphistory-0.0.3: libvdr-zaphistory.so.1.3.41.
diff -ru zaphistory-0.0.3-orig/zaphistoryosd.c zaphistory-0.0.3/zaphistoryosd.c
--- zaphistory-0.0.3-orig/zaphistoryosd.c	2006-03-10 21:49:10.415716000 +0100
+++ zaphistory-0.0.3/zaphistoryosd.c	2006-03-10 21:50:16.175825750 +0100
@@ -217,7 +217,11 @@
     const cSchedules *schedules = cSchedules::Schedules( *schedLock );
     const cSchedule *schedule = schedules->GetSchedule( channel );
       
+#if (VDRVERSNUM < 10344)
+    return AddSubMenu( new cMenuEvent(schedule->GetEventAround(time(NULL)), true) );
+#else
     return AddSubMenu( new cMenuEvent(schedule->GetEventAround(time(NULL)), true, true) );
+#endif
 }
 
 eOSState cMenuZappedChannels::ProcessKey(eKeys Key)
Nur in zaphistory-0.0.3: zaphistoryosd.o.
